Formula/c/cycode.rb

```diff
@@ -19,7 +19,6 @@ class Cycode < Formula
 
   depends_on "maturin" => :build # for mcp
   depends_on "pkgconf" => :build
-  depends_on "rust" => :build # for mcp, upstream bug report, https://github.com/PyO3/maturin/issues/2642
```

Comment:
This means we're letting maturin download it's own rust, we should use our own no?